conf/locale/babel_underscore.cfg

```diff
@@ -1,9 +1,5 @@
 # Use this configuration file for extracting strings from .underscore files.
-[underscore: cms/templates/**.underscore]
-input_encoding = utf-8
-[underscore: common/templates/**.underscore]
-input_encoding = utf-8
-[underscore: lms/templates/**.underscore]
+[underscore: **.underscore]
```

Comment:
Good question. Yes, there seem to be quite a few:

    cms/static/coffee/fixtures
    cms/templates/js/mock
    common/lib/xmodule/xmodule/js/fixtures
    lms/static/js/fixtures

Some of these do include ``gettext`` calls although they really shouldn't. It is a shame we can't blacklist certain directories, because getting a positive set of directories could be tricky.

I suppose it should the rules should be similar to the mako version. Something like this:

   cms/templates/**.underscore
   common/static/common/templates/**/**.underscore
   common/templates/**.underscore
   lms/templates/**.underscore
    */djangoapps/*/static/*/templates/*.underscore